spec/unit/cli_spec.rb

```diff
@@ -530,6 +530,13 @@
 
       run_command(["remove", "foo"])
     end
+
+    it "triggers the remove task with --all option if given" do
+      expect_any_instance_of(RemoveTask).to receive(:remove).
+        with(an_instance_of(SystemDescriptionStore), anything(), verbose:false, all:true)
```

Comment:
Do not use parentheses for method calls with no arguments.<br>Space missing after colon.